src/api/players/server/getPlayerLicense.lua

```diff
@@ -3,10 +3,10 @@
 ---@return string
 ---@public
 function _NCS:getPlayerLicense(playerId)
-    for k, v in pairs(GetPlayerIdentifiers(playerId)) do
+    for _, v in pairs(GetPlayerIdentifiers(playerId)) do
         if (string.sub(v, 1, string.len("license:")) == "license:") then
-            return (v)
+            return v
         end
     end
-    return (false)
+    return false
 end
```

Comment:
The paranthesis are part of the conventions I imposed for the project.

Reply:
> The paranthesis are part of the conventions I imposed for the project.

Should be good now